massgov/mayflower
# DP-12404 Add blocks twig to some templates

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Needed to add a block to the following twig templates: executive-order.twig, policy-advisory.twig,  court-rules.twig to make a change in what document links are being used on the Drupal side. 

## Related Issue / Ticket

- [JIRA issue DP-12404](https://jira.mass.gov/browse/DP-12404)
- [Github issue]()

## Steps to Test


1. View the following pages to make sure no changes in design around document link section vs mayflower.digital.mass.gov:
   - [ ] Executive order `/?p=pages-executive-order`
   - [ ] Policy Advisory Directive `/?p=pages-policy-advisory-directive` and `/?p=templates-policy-advisory`
   - [ ] Rules of court `/?p=pages-rules-of-court` and `/?p=templates-court-rules`

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
